src/main/webapp/assets/js/feedScript.js

```diff
@@ -573,6 +629,43 @@ function closeModal() {
   }
 }
 
+/**
+ * On click of the submit button, sends modal data to the servlet.
+ * @return {void}
+ */
+async function submitModal() {
+  const collegeId = (new URLSearchParams(window.location.search)).get('collegeid');
+  console.log('found id: ' + collegeId);
```

Comment:
Do we want to remove this print before merging? Assuming it's for debugging purposes

Reply:
ah yes thank you